src/js/components/ArtefactCanvas.vue

```diff
@@ -44,15 +44,15 @@
 </template>
 
 <script>
-/*
- * The mask seems to dilate with every edit.
- * Check for fix, perhaps something to do with
- * potrace algorithm, or perhaps the vector path
- * stroke makes the path a bit larger with every
- * draw.
- */
-import { trace } from '../utils/Potrace.js'
-import { clipCanvas } from '../utils/VectorFactory'
+import { trace } from '~/utils/Potrace.js'
```

Comment:
I'm afraid I'm not so up to speed with all this svg and geo stuff going on—do you mind adding some comments to explain what the various methods attempt to accomplish?